### PR TITLE
Add info about Good First Issues on GitHub

### PR DESCRIPTION
**Pull request by tapaswenipathak**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/master/CONTRIBUTING.md)?
- [X] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----
Fixes https://github.com/github/opensource.guide/issues/1052.

### CODE REVIEW

File: _articles/best-practices.md [hunk 1]
```diff
@@ -82,6 +82,8 @@ If you meet with other maintainers, or make a major decision in private, documen
 
 That way, anybody who joins your community will have access to the same information as someone who's been there for years.
 
+As a open source maintainer you would like being respectful and encouraging for open source contributors. People contributing in an open source project is a gain for everyone stakeholder involved, be you as a maintainer or the one is contributing. For helping you in the endeavour we have good first issues, if you label issues as good first issues we will show the issues as a GitHub feature.
```
Comment (mlinksva) [inline _articles/best-practices.md]: I'd drop this paragraph. The rest of this section is higher level and I don't think good first issues is better in the building community article.

File: _articles/building-community.md [hunk 2]
```diff
@@ -32,6 +32,7 @@ Start with your documentation:
 
 * **Make it easy for someone to use your project.** [A friendly README](../starting-a-project/#writing-a-readme) and clear code examples will make it easier for anyone who lands on your project to get started.
 * **Clearly explain how to contribute**, using [your CONTRIBUTING file](../starting-a-project/#writing-your-contributing-guidelines) and keeping your issues up-to-date.
+* **Beginner friendly issues**, as a open source maintainer you should create few beginner friendly tickets for people who have never contributed in the open source project or say any open source project, you can label such issues as good first issues. You should encourage contributors for resolving such issues before they pick more severe issues during the journey of working with an open source project.
```
Comment (mlinksva) [inline _articles/building-community.md]: I would rather not recommend synthetically creating beginner friendly tickets, rather labeling any that exist/get created. Also I'm not sure about the last sentence. This bullet could be one brief sentence, and link to https://help.github.com/en/articles/helping-new-contributors-find-your-project-with-labels

File: _articles/how-to-contribute.md [hunk 3]
```diff
@@ -216,6 +216,8 @@ Open source isn't an exclusive club; it's made by people just like you. "Open so
 
 You might scan a README and find a broken link or a typo. Or you're a new user and you noticed something is broken, or an issue that you think should really be in the documentation. Instead of ignoring it and moving on, or asking someone else to fix it, see whether you can help out by pitching in. That's what open source is all about!
 
+Good first issue is a feature GitHub ships for encouraging new and beginner open source contributors for contributing in an open source project. If you are new for an open sourcr project or have never contributed in an any open source projects you can write a fix for a good first issues. These tickets helps you learn the open source organisation culture, maintainers ethics and communication practices. The maintainers keep the thought in mind that you are a new and beginner contributor in open source project if you are taking a good first issue.
```
Comment (mlinksva) [inline _articles/how-to-contribute.md]: I'd suggest a much tighter paragraph and probably talk about `/contribute` as mentioned in https://github.com/github/opensource.guide/issues/1052 as the previous paragraph is about looking for stuff to work on in a specific repo.

**Comment by mlinksva** [comment]
Thanks for opening this @tapaswenipathak! I'm going to suggest a review from @tiferet who opened #1052 and will probably have more concrete feedback/suggestions.

File: _articles/best-practices.md (hunk 1 above)
Comment (tiferet) [reply _articles/best-practices.md]: I'm not sure this is the correct place for this. I suggest adding a paragraph in the section `Share the workload`, right after the sentence "If you're looking for others to pitch in, start by asking around." The new paragraph could be something like this:

"One way to gain new contributors is to explicitly [label issues that are simple enough for beginners to tackle](https://help.github.com/en/articles/helping-new-contributors-find-your-project-with-labels). GitHub will then surface these issues in various places on the platform, increasing their visibility."

File: _articles/building-community.md (hunk 2 above)
Comment (tiferet) [reply _articles/building-community.md]: I agree with Mike. See my suggested change.

Comment (tiferet) [inline _articles/building-community.md]:
```suggestion
* **Good first issues**: To help new contributors get started, consider explicitly [labeling issues that are simple enough for beginners to tackle](https://help.github.com/en/articles/helping-new-contributors-find-your-project-with-labels). GitHub will then surface these issues in various places on the platform, increasing useful contributions, and reducing friction from users tackling issues that are too hard for their level.
```

File: _articles/how-to-contribute.md (hunk 3 above)
Comment (tiferet) [reply _articles/how-to-contribute.md]: Again I agree with Mike. In addition, this paragraph should go after the quotation. See my suggestion below.

File: _articles/how-to-contribute.md [hunk 4]
```diff
@@ -216,6 +216,8 @@ Open source isn't an exclusive club; it's made by people just like you. "Open so
...
+
 > [28% of casual contributions](https://www.igor.pro.br/publica/papers/saner2016.pdf) to open source are documentation, such as a typo fix, reformatting, or writing a translation.
```
Comment (tiferet) [inline _articles/how-to-contribute.md]:
```suggestion

If you're looking for existing issues you can fix, every open source project has a `/contribute` page that highlights beginner-friendly issues you can start out with. Navigate to the main page of the repository on GitHub, and add `/contribute` at the end of the URL (for example `https://github.com/facebook/react/contribute`).

```

**Comment by tapaswenipathak** [comment]
Hey folks: Done per review.

File: _articles/how-to-contribute.md [hunk 5]
```diff
@@ -216,8 +216,12 @@ Open source isn't an exclusive club; it's made by people just like you. "Open so
 
 You might scan a README and find a broken link or a typo. Or you're a new user and you noticed something is broken, or an issue that you think should really be in the documentation. Instead of ignoring it and moving on, or asking someone else to fix it, see whether you can help out by pitching in. That's what open source is all about!
 
+If you are a beginner in open source you can work on a good first issue. Good first issue is a feature GitHub ships for encouraging new and begineer open source contributors for contributing in an open source project. You can find such issues in [contribute pages](https://github.com/prettier/prettier/contribute).
```
Comment (tiferet) [inline _articles/how-to-contribute.md]: It looks like you forgot to delete this.

Comment (tapaswenipathak) [reply _articles/how-to-contribute.md]: Ref https://github.com/github/opensource.guide/pull/1110#discussion_r327692279.

File: _articles/best-practices.md (hunk 1 above)
Comment (tapaswenipathak) [reply _articles/best-practices.md]: 0ffac237dab08e572552f11fc7eeb3d717913172.

File: _articles/how-to-contribute.md (hunk 5 above)
Comment (tiferet) [reply _articles/how-to-contribute.md]: I'm not sure what you're referring to in that comment. You added the paragraph I suggested below, but didn't delete this one.

Comment (tapaswenipathak) [reply _articles/how-to-contribute.md]: I see, right, misunderstanding in continuity. Done.

File: _articles/how-to-contribute.md [hunk 6]
```diff
@@ -218,6 +218,8 @@ You might scan a README and find a broken link or a typo. Or you're a new user a
 
 > [28% of casual contributions](https://www.igor.pro.br/publica/papers/saner2016.pdf) to open source are documentation, such as a typo fix, reformatting, or writing a translation.
 
+If you're looking for existing issues you can fix, every open source project has a `/contribute` page that highlights beginner-friendly issues you can start out with. Navigate to the main page of the repository on GitHub, and add `/contribute` at the end of the URL (for example `https://github.com/facebook/react/contribute`).
```
Comment (mlinksva) [inline _articles/how-to-contribute.md]:
```suggestion
If you're looking for existing issues you can fix, every open source project has a `/contribute` page that highlights beginner-friendly issues you can start out with. Navigate to the main page of the repository on GitHub, and add `/contribute` at the end of the URL (for example [`https://github.com/facebook/react/contribute`](https://github.com/facebook/react/contribute)).
```

Suggest making the example a link that people can easily see by clicking on.

**Pull request merged**